src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -6,3 +6,96 @@
 from azure.cli.help_files import helps #pylint: disable=unused-import
 
 #pylint: disable=line-too-long
+
+helps['network dns record-set create'] = """
```

Comment:
in this case the text all starts with "gets or sets the xyz value", so I opted for overwriting.  If we're ok with some strange help artifacts, then grabbing the __doc__ works fine here.  To not incur more help debt, I added overrides with better text

---
In reply to: [75556265](https://github.com/Azure/azure-cli/pull/718#discussion_r75556265) [](ancestors = 75556265)